### PR TITLE
add notice about CAPI to credential docs

### PR DESCRIPTION
**Pull request by anvddriesch**
### What does this PR do?

Towards https://github.com/giantswarm/giantswarm/issues/29231
The document seems up to date for vintage. However, there is no mention of CAPI at all and I could not see other documentation on how to deal with cloud provider credentials in CAPI within our docs.

For now I added a link to upstream documentation. 

### What does it look like?

(If this is more than a textual change, a screenshot can help in some cases to speed up the review process.)

### Any background context you can provide?

(Please link public issues or summarize if not public.)

### Have you maintained the front matter?

(Please bump the last_review_date in case this qualifies as a review for an entire page. Provide user_questions which should be answered in the page. Provide a meaningful description.)

### CODE REVIEW

**Comment by calvix** [comment]
I would say using our docs is better than the public which are not exactly correct for our use case.

https://docs.giantswarm.io/getting-started/cloud-provider-accounts/cluster-api/aws/
https://docs.giantswarm.io/getting-started/cloud-provider-accounts/cluster-api/azure/

**Comment by anvddriesch** [comment]
@calvix :D Sweet, that's exactly what I was looking for and somehow didn't see. I will link those.

**Review by pipo02mix — APPROVED** [review]
LGTM

File: src/content/use-the-api/management-api/credentials/index.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ user_questions:
   - How can I set cloud provider credentials via the Management API?
 ---
 
+### Notice: This document describes how to configure cloud provider credentials for _vintage_ Giant Swarm Clusters. For _CAPI_ Clusters, documentation is available for [CAPA](https://docs.giantswarm.io/getting-started/cloud-provider-accounts/cluster-api/aws/) and [CAPZ](https://docs.giantswarm.io/getting-started/cloud-provider-accounts/cluster-api/azure/)
```
Comment (AndiDog) [inline src/content/use-the-api/management-api/credentials/index.md:20]:
```suggestion
### Notice: This document describes how to configure cloud provider credentials for _vintage_ Giant Swarm Clusters. For _CAPI_ Clusters, documentation is available for [CAPA]({{< relref "/getting-started/cloud-provider-accounts/cluster-api/aws/" >}}) and [CAPZ]({{< relref "/getting-started/cloud-provider-accounts/cluster-api/azure/" >}})
```

**Review by tuladhar — APPROVED** [review]

**Review by ssyno — APPROVED** [review]

**Comment by AndiDog** [comment]
I'll get the build fixed and PR merged. `relref` is picky about ending with or without slash...

**Comment by anvddriesch** [comment]
@AndiDog thank you :smile:

File: src/content/use-the-api/management-api/credentials/index.md [hunk 2]
```diff
@@ -17,6 +17,8 @@ user_questions:
   - How can I set cloud provider credentials via the Management API?
 ---
 
+### Notice: This document describes how to configure cloud provider credentials for _vintage_ Giant Swarm Clusters. For _CAPI_ Clusters, documentation is available for [CAPA]({{< relref "/getting-started/cloud-provider-accounts/cluster-api/aws" >}}) and [CAPZ]({{< relref "/getting-started/cloud-provider-accounts/cluster-api/azure" >}})
+
```
Comment (marians) [inline src/content/use-the-api/management-api/credentials/index.md:21]: Nitpick: "clusters" should not be capitalized.

**Pull request merged**